python/foxglove-sdk-examples/ws-services/main.py

```diff
@@ -0,0 +1,125 @@
+"""
+This example demonstrates how to use the Foxglove WebSocket API to implement services which can be
+called from the Service Call panel in the Foxglove app.
+
+https://docs.foxglove.dev/docs/visualization/panels/service-call
+"""
+
+import argparse
+import json
+import logging
+
+from foxglove import (
+    Service,
+    ServiceSchema,
+    start_server,
+    Capability,
+    Client,
+    ServiceHandler,
+)
+
+
+def logging_handler(
+    service_name: str, client: Client, call_id: int, encoding: str, payload: bytes
```

Comment:
The problem with offering this API is it's not possible to use async Python or do work in another thread.

It's nice and simple for simple services, but it makes more complex ones impossible.

The responder object pattern we used in rust allows for all the above, while being a little more complex to use for simple cases.

Do we offer two distinct APIs or should we try to offer one way of doing it? I'm in favor of the latter (it's actually part of the zen of Python that there should ideally be one and only one obvious way to do it)